src/views/ModelPlan/TaskList/_components/TaskListStatus/index.tsx

```diff
@@ -57,19 +57,17 @@ const TaskListStatus = ({
             </p>
           )}
 
-          <div>
-            <UswdsReactLink
-              to={
-                readOnly
-                  ? `/models/${modelID}/task-list`
-                  : `/models/${modelID}/status`
-              }
-              className="display-flex flex-align-center"
-            >
-              {icon && <IconEdit className="margin-right-1" />}
-              {updateLabel && t('update')}
-            </UswdsReactLink>
-          </div>
+          {!readOnly && (
+            <div>
+              <UswdsReactLink
+                to={`/models/${modelID}/status`}
+                className="display-flex flex-align-center"
+              >
+                {icon && <IconEdit className="margin-right-1" />}
+                {updateLabel && t('update')}
+              </UswdsReactLink>
+            </div>
+          )}
```

Comment:
Since the props `icon` and `updateLabel` are now always dependent and tied to `readonly`, i don't think they are needed anymore.  The link to update only exists on the task list.  I'm fine with possibility leaving this in for a future flexibility, but as of now with this change, they don't really do anything.